packages/trader/src/AppV2/Components/TradeParameters/TakeProfit/take-profit.scss

```diff
@@ -0,0 +1,42 @@
+$HANDLEBAR_HEIGHT: var(--core-size-1000);
+$HEADER_TITLE_HEIGHT: var(--core-size-3200);
+$FOOTER_BUTTON_HEIGHT: var(--core-size-4000);
+
+.take-profit {
+    &__wrapper {
+        position: relative;
+        height: calc(400px - $HANDLEBAR_HEIGHT - $HEADER_TITLE_HEIGHT - $FOOTER_BUTTON_HEIGHT);
+        padding-block-start: 0;
+
+        &--definition {
+            height: calc(400px - $HANDLEBAR_HEIGHT - $HEADER_TITLE_HEIGHT);
+            padding-block-start: 0;
+        }
+    }
+
+    &__warning {
+        margin-block-start: var(--core-spacing-800);
+    }
+
+    &__content {
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+        padding-block: var(--core-spacing-400);
+    }
+
+    &__overlay {
+        all: unset;
+        height: var(--core-size-2800);
+        width: calc(100vw - (2 * var(--core-spacing-800)));
```

Comment:
Why multiplying by 2 ? can we use var(--core-spacing-1600)

Reply:
I was trying to show that it's double margin (left and right), but yes, better to remove it and leave var(--core-spacing-1600)